app/main.py

```diff
@@ -2,5 +2,14 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results_cache = {}
+
+    def inner(*args) -> dict:
```

Comment:
Add here type hint for args please :) And you not  returning dict , it can be any type ( int , float etc. )

Reply:
> Add here type hint for args please :) And you not returning dict , it can be any type ( int , float etc. )

Reply:
Yes, of course, but I was just testing under tests and wanted to get rid of "Any" to lighten the code while keeping functionality